lib/inspec/resources/podman_container.rb

```diff
@@ -0,0 +1,84 @@
+require "inspec/resources/podman"
+require_relative "podman_object"
+
+# Change module if required
+module Inspec::Resources
+  class PodmanContainer < Inspec.resource(1)
+    include Inspec::Resources::PodmanObject
+    name "podman_container"
+    supports platform: "unix"
+
+    desc "A resource to retrieve information about podman container"
+
+    example <<~EXAMPLE
+        describe podman_container("sweet_mendeleev") do
+          it { should exist }
+          it { should be_running }
+          its("id") { should eq "591270d8d80d26671fd6ed622f367fbe19004d16e3b519c292313feb5f22e7f7" }
+          its("image") { should eq "docker.io/library/nginx:latest" }
+          its("labels") { should include "maintainer"=>"NGINX Docker Maintainers <docker-maint@nginx.com>" }
+          its("ports") { should eq nil }
+        end
+
+        describe podman_container(id: "591270d8d80d2667") do
+          it { should exist }
+          it { should be_running }
+        end
+    EXAMPLE
+
+    def initialize(opts = {})
+      skip_resource "The `podman_container` resource is not yet available on your OS." unless inspec.os.unix?
```

Comment:
This line might not work, if we do not use `raise Inspec::Exceptions::ResourceSkipped` somewhere in this resource..just have a check.

Reply:
This works. We don't need to raise exception here.